demos/ErrorEstimation/FloatSum.cpp

```diff
@@ -0,0 +1,133 @@
+//--------------------------------------------------------------------*- C++ -*-
+// clad - The C++ Clang-based Automatic Differentiator
+//
+// A demo demonstrating the usage of clad's error estimation capabilities.
+//
+// author:  Garima Singh
+//----------------------------------------------------------------------------//
+
+// To run the demo please type:
+// path/to/clang  -Xclang -add-plugin -Xclang clad -Xclang -load -Xclang \
+// path/to/libclad.so  -I../include/ -x c++ -std=c++11 FloatSum.cpp
+//
+// A typical invocation would be:
+// ../../../../obj/Debug+Asserts/bin/clang  -Xclang -add-plugin -Xclang clad \
+// -Xclang -load -Xclang ../../../../obj/Debug+Asserts/lib/libclad.dylib     \
+// -I../include/ -x c++ -std=c++11 FloatSum.cpp
+//
+// To plot the results install gnuplot and type:
+//
+// 1) The following if you want a comparision between calculated reuslts
+//
+// NOTE: By default clad calculates the error estimates pessimistically,
+// which means that all estimates generated by clad reperesent the worst case
+// error in all floating point operations. Hence, the in-built model gives an
+// upperbound to the error in the function rather than a close estimate.
+// To know how you can achieve closer estimates, you may checkout the
+// "CustomModel" demo or Clad tutorials.
+//
+// gnuplot -e "plot 'actualError.dat' using 1:2 with lines; replot \
+// 'estimateError.dat' using 1:2 with lines; pause -1"
+//
+// 2) The following if you want to visualize the difference between
+// float and fixed results
+//
+// gnuplot -e "plot 'kahan.dat' using 1:2 with lines; replot 'vanilla.dat' \
+// using 1:2 with lines; pause -1"
+
+// Necessary for clad to work include
+#include "clad/Differentiator/Differentiator.h"
+
+#include <iostream>
+// Necessary for saving data to plot.
```

Comment:
```suggestion
#include <iostream> // Necessary for saving data to plot.
```
That way we do not risk somebody to insert an include and invalidate the comment.